chaos_kitten/cli.py

```
@@ -175,371 +174,84 @@ def scan(
-            
-            if total_critical > 0:
-                console.print(f"[bold red]❌ Found {total_critical} critical issue(s). Failing pipeline.[/bold red]")
+            critical_vulns = [v for v in vulnerabilities if str(v.get("severity", "")).lower() == "critical"]
+            if critical_vulns:
+                console.print(f"[bold red]❌ {len(critical_vulns)} critical vulnerabilities. Failing pipeline.[/bold red]")
                 raise typer.Exit(code=1)
-
-    except typer.Exit:
-        raise
+        
     except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)